person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, parent_permission: true, name: 'Unknow')
+    @id = rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if @age >= 18 || @parent_permission == true
+
+    false
+  end
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think we can make this code more concise and clear. Refactoring this method to something like in the snippet below works as well:

  ```
  def of_age?
    @age >= 18
  end
  ```